lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -5,6 +5,8 @@ package DDG::Spice::Cryptocurrency;
 use DDG::Spice;
 with 'DDG::SpiceRole::NumberStyler';
 
+use Data::Dumper;
```

Comment:
@pjhampton for what it's worth, `Data::Printer` is already added to all IA classes in the DDG module. You can simply use `p($foo)` to log structures 😄